interfaces/AW-App/src/app/services/programs-service-api.service.ts

```diff
@@ -101,7 +101,7 @@ export class ProgramsServiceApiService {
     return this.apiService
       .post(
         environment.url_121_service_api,
-        '/sovrin/create-connection/custom-data/overwrite',
+        '/sovrin/create-connection/custom-data',
```

Comment:
Was this something that was 'forgotten' earlier?

I can see no real changes to this in this PR, is that correct?